README.md

```diff
@@ -79,22 +79,28 @@ These steps are completed by the admins of this repo. They might ask questions o
+3. IBM Cloud will automatically rebuild the [certifications-staging site](https://certifications-staging-openlibertyio.mqj6zf7jocq.us-south.codeengine.appdomain.cloud/certifications) and [staging site](https://staging-openlibertyio.mqj6zf7jocq.us-south.codeengine.appdomain.cloud/certifications). If you have access, you can track the progress in the [Slack channel](https://app.slack.com/client/T15GKHBT4/C01GX9P8YP2).
 
-5. When the build has finished, check to make sure the page render correctly on the [certifications-staging site](https://certifications-staging-openlibertyio.mqj6zf7jocq.us-south.codeengine.appdomain.cloud/certifications) or [staging site](https://staging-openlibertyio.mqj6zf7jocq.us-south.codeengine.appdomain.cloud/certifications). The latter includes the entire site, while the former just has the /certifications/ content.  If you need to verify links to other parts of the site (outside of the /certifications/ content) then you'll need to wait for the full [staging site](https://staging-openlibertyio.mqj6zf7jocq.us-south.codeengine.appdomain.cloud/) to build.
+4. When the build has finished, check to make sure the page renders correctly on the [certifications-staging site](https://certifications-staging-openlibertyio.mqj6zf7jocq.us-south.codeengine.appdomain.cloud/certifications) or [staging site](https://staging-openlibertyio.mqj6zf7jocq.us-south.codeengine.appdomain.cloud/certifications). The latter includes the entire site, while the former just has the /certifications/ content.  If you need to verify links to other parts of the site (outside of the /certifications/ content) then you'll need to wait for the full [staging site](https://staging-openlibertyio.mqj6zf7jocq.us-south.codeengine.appdomain.cloud/) to build.
 
    This is the final check before the page is published live on the [production site](https://openliberty.io/).
 
    If there are any problems found on the [staging site](https://staging-openlibertyio.mqj6zf7jocq.us-south.codeengine.appdomain.cloud/), you must resolve them quickly or revert the PR.
    
    Make any changes in the author's branch, and push to both `draft` and `staging`.
    
-6. To publish the content, create a PR from `staging` branch to `prod` branch and add the [Release Architect](https://github.com/orgs/OpenLiberty/teams/release-architect) (or other admin) as approver.
+5. To publish the content, create a PR from `staging` branch to `prod` branch and add the [Release Architect](https://github.com/orgs/OpenLiberty/teams/release-architect) (or other admin) as approver.
 
-7. When the PR is approved, merge it into `prod`.
+   - Note: Anyone can create this PR, it does not have to be the Release Architect or an admin.
```

Comment:
If the goal is to get content creators like myself to create these PRs, I don't think this change works at a functional level, as it's buried deep in a section that starts with:
```
Admins: Editing and Publishing TCK Results
These steps are completed by the admins of this repo. They might ask questions or make suggestions regarding the content. They might also make edits directly to the file before publishing.
```

No content creator is actually going to read through this section at all because it doesn't pertain to them.